deeplearning4j-keras/src/test/java/org/deeplearning4j/keras/DeepLearning4jEntryPointTest.java

```diff
@@ -0,0 +1,49 @@
+package org.deeplearning4j.keras;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.io.File;
+import java.io.IOException;
+
+public class DeepLearning4jEntryPointTest {
+
+    private DeepLearning4jEntryPoint deepLearning4jEntryPoint = new DeepLearning4jEntryPoint();
+
+    public @Rule ExpectedException thrown = ExpectedException.none();
+
+    @Test
+    public void shouldFitTheSampleSequentialModel() throws Exception {
+        // Given
+        File model = prepareResource("theano_mnist/model.h5");
+        File features = prepareResource("theano_mnist/x_train.h5");
+        File labels = prepareResource("theano_mnist/y_train.h5");
+
+        EntryPointFitParameters entryPointParameters = EntryPointFitParameters
```

Comment:
Comment on overall api. Is there any way you can just use a neuralnet configuration directly or better yet leverage the work @turambar and @eraly are doing with model import?